src/components/CardHeaderComment.tsx

```diff
@@ -0,0 +1,43 @@
+import { nullable, Link, Text } from '@taskany/bricks';
+import { gapXs, gray8 } from '@taskany/colors';
+import styled from 'styled-components';
+
+interface CardHeaderProps extends React.HTMLAttributes<HTMLDivElement> {
+    name?: string | null;
```

Comment:
does it make sense to render comment w/o author name?

Reply:
No, it doesn't make sense.